src/Resources/config/listener.yml

```diff
@@ -6,6 +6,7 @@
 #
 # - 30: ContainerScopeListener
 # - 28: RefererIdListener
+# - 27: InsecureInstallationListener
```

Comment:
I do not really understand this. Why should there be a number between if it is change in every PR that needs to. I think it would be better to do something like that: 

```
- 300: ContainerScopeListener
- 280: RefererIdListener
- 260: InsecureInstallationListener
- 240: InitializeSystemListener
- 220: ToggleViewListener
- 200: OutputFromCacheListener
```

In this case you could rework the priority as a refactoring and not in PR itself ... Won´t you think this is better? In most reason changing the priority of all other listeners is not really in scope of a PR.